app/main/controller/ccp/sso.php

```diff
@@ -134,7 +134,7 @@ public function requestAuthorization($f3){
 
             $urlParams = [
                 'response_type' => 'code',
-                'redirect_uri' => Controller\Controller::getEnvironmentData('URL') . $f3->build('/sso/callbackAuthorization'),
+                'redirect_uri' => Controller\Controller::getEnvironmentData('URL') . Controller\Controller::getEnvironmentData('BASE') . $f3->build('/sso/callbackAuthorization'),
```

Comment:
if I add this, the redirect URL will be www.[domain]/pathfinder/pathfinder/sso/...

Reply:
I have no clue how this got added, but it broke my call to the SSO